doc/plugin_agent_workloadattestor_systemd.md

```diff
@@ -0,0 +1,23 @@
+# Agent plugin: WorkloadAttestor "systemd"
+
+The `systemd` plugin generates selectors based on [systemd](https://systemd.io/) unit properties of the workloads calling the agent.
+
+This plugin does not accept any configuration options.
+
+General selectors:
+
+| Selector               | Value                                                                                                               |
+|------------------------|---------------------------------------------------------------------------------------------------------------------|
+| `systemd:Id`           | The unit Id of the workload (e.g. `systemd:Id:nginx.service`)                                                       |
```

Comment:
I think this selector would be more clear if it was `system:pid`, as it's the ID of the process and the method to get the unit is `GetUnitByPID`. 

WDYT?

Reply:
No, this is not the process ID, its the systemd unit Id: for e.g. `nginx.service`. IMO, the PID will not be very useful as a selector because it is not a stable attribute.